scripts/assets/maven_dependencies.textproto

```diff
@@ -0,0 +1,875 @@
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.room:room-runtime:2.2.5"
+  artifact_version: "2.2.5"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.vectordrawable:vectordrawable-animated:1.1.0"
+  artifact_version: "1.1.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.viewpager2:viewpager2:1.0.0"
+  artifact_version: "1.0.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.viewpager:viewpager:1.0.0"
+  artifact_version: "1.0.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.work:work-runtime-ktx:2.4.0"
+  artifact_version: "2.4.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "androidx.work:work-runtime:2.4.0"
+  artifact_version: "2.4.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "com.crashlytics.sdk.android:crashlytics:2.9.8"
+  artifact_version: "2.9.8"
+  license {
+    license_name: "Crashlytics Terms of Service"
+    original_link: "https://try.crashlytics.com/terms/terms-of-service.pdf"
+    direct_link_only {
+      url: "https://firebase.google.com/terms/crashlytics"
+    }
+    is_original_link_invalid: true
+  }
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
+    scrapable_link {
+      url: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
```

Comment:
Rather than pointing to ``master``, we should point to the specific commit corresponding to the version that we depend on to guard against the license changing without us updating the version.

Ditto for other raw github retrievals below (except when they come from our repo since we version those files).

Reply:
I tried this but I think it's not possible, I couldn't obtain a link to the specific commit.

Reply:
This is possible. Here are the steps to find this:

1. Navigate to the license file on GitHub (e.g. https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE)
2. Navigate to the repo link version (e.g. https://github.com/firebase/firebase-android-sdk/blob/master/LICENSE)
3. Click the three dots on the upper right and click 'copy permalink' which should give you something like this: https://github.com/firebase/firebase-android-sdk/blob/52e1cf89efd795d798ae5485b8503e04e4a936b8/LICENSE
4. Verify that this is the correct license for the version that we're using (tying to specific version isn't as important as long as we have a frozen permalink)
5. Convert the link back to a raw link: https://raw.githubusercontent.com/firebase/firebase-android-sdk/52e1cf89efd795d798ae5485b8503e04e4a936b8/LICENSE
6. Use this raw link in files

Reply:
Done.

Reply:
This was an example--all the other GitHub references still need to be made into permalinks.

Reply:
Done.

Reply:
I think "com.crashlytics.sdk.android:crashlytics:2.9.8" also needs to be updated, but the rest look correct.

Reply:
Thanks, updated.